lib/preprocessor.js

```diff
@@ -121,7 +130,8 @@ var createPreprocessor = function (config, basePath, injector) {
 
         nextPreprocessor(null, thisFileIsBinary ? buffer : buffer.toString())
       })
-    })
+    }
+    return fs.readFile(file.originalPath, handleFile)
```

Comment:
fs.readFile is the correct call as handleFile was the original anonymous callback at line 79

I've renamed handleFile to readFileCallback to make it clearer